app/controllers/mixins/compressed_ids.rb

```diff
@@ -1,4 +1,5 @@
 module CompressedIds
+  CID_REGEXP = '[0-9]r?[0-9]*'.freeze
   #
   # Methods to convert record id (id, fixnum, 12000000000056) to/from compressed id (cid, string, "12c56")
```

Comment:
is 12c56 a valid compressed id in the comment ?  isn't it only with the r as per the regexp ?